src/format.js

```diff
@@ -0,0 +1,105 @@
+document.addEventListener('DOMContentLoaded', () => {
+  const STRIKE = 10;
+  const STRIKE_SYMBOL = 'X';
+  const SPARE_SYMBOL = '/';
+  let isParty = true;
+  let game = new Game(Frame);
+
+  let setTotal = () => {
+    document.getElementById('currentScore').innerHTML = game.score();
+  };
+
+  let individualScores = () => {
+    let collumnID = 1;
+    let scores = game.runningScores();
+    scores.forEach((score) => {
+      document.getElementById(`scoreFrame${collumnID}`).innerHTML = `${score}`;
+      collumnID += 1;
+    });
+  };
+
+  let colourGen = () => {
+      let a = Math.random() * 255;
+      let b = Math.random() * 255;
+      let c = Math.random() * 255;
+   return "rgb(" + a + ", " + b + ", " + c + ")"
+}
+
+function colours() {
+  document.getElementById('scorecardLabels').style.backgroundColor = colourGen();
+  document.getElementById('scoreCardScores').style.backgroundColor = colourGen();
+  document.getElementById('frameResults').style.backgroundColor = colourGen();
+  document.body.style.backgroundColor = colourGen();
+  document.body.style.color = colourGen();
+}
+
+  // this logic should be in model not in the formatting page...
+
+  let strikeConverter = rolls => rolls.map(roll => (roll === STRIKE ? STRIKE_SYMBOL : roll));
+
+  let spareConverter = (rolls) => {
+    if (rolls[0] + rolls[1] === STRIKE) { rolls[1] = SPARE_SYMBOL; }
+    if (rolls[1] + rolls[2] === STRIKE) { rolls[2] = SPARE_SYMBOL; }
+    return rolls;
+  };
+
+  let rollPrinter = game => game.map((frame) => {
+    let view = strikeConverter(frame);
+    view = spareConverter(view);
+    return view.join('');
+  });
+
+  // this logic is to be refactored out
+
+  let showRolls = () => {
+    let collumnID = 1;
+    const frames = rollPrinter(game.view());
+    frames.forEach((frame) => {
+      document.getElementById(`views${collumnID}`).innerHTML = `${frame}`;
+      collumnID += 1;
+    });
+  };
+
+  let roll = (value) => { game.play(value); setTotal(); individualScores(); showRolls(); colours();};
+
+  let rollButton = (pins) => {
+    const btn = document.createElement('button');
+    btn.id = pins;
+    btn.onclick = () => { roll(pins); };
+    btn.innerHTML = pins;
+    return btn;
+  };
+
+  let showScoreButtons = (max = STRIKE) => {
+    for (let i = 0; i <= max; i += 1) {
+      const btn = rollButton(i);
+      document.getElementById('changeScores').appendChild(btn);
+    }
+  };
+
+
+  let createFieldWithID = (id) => {
+    const field = document.createElement('td');
+    field.id = id;
+    return field;
+  };
+
+  let createScoreCard = () => {
```

Comment:
I would tend to write functions using the standard 

```
function createScoreCard() {
  ..
}
```

notation. The lambda style notation (I'm actually not sure if they are called lambda's in javascript, that's a C#ism) is used more with callbacks - they are a bit like Ruby code blocks. This code feels confusing to me because it looks like it's a very long method when really it's a set of function declarations.